### PR TITLE
Add ValidateAddress to stakepoold RPC

### PR DESCRIPTION
**Pull request by jholdstock**
This work is on top of #403, #402 and #395. Marking the PR as a draft, and I will rebase and request a review once the previous three are merged.

More work towards #227

### CODE REVIEW

**Comment by jholdstock** [comment]
Rebased and ready for a review.

File: stakepooldclient/stakepooldclient.go [hunk 1]
```diff
@@ -3,6 +3,7 @@ package stakepooldclient
 import (
 	"errors"
 	"fmt"
+	"github.com/decred/dcrd/dcrutil"
```
Comment (chappjc) [inline stakepooldclient/stakepooldclient.go]: imports need ordering (separate std lib from others)

File: stakepooldclient/stakepooldclient.go [hunk 2]
```diff
@@ -265,6 +266,48 @@ func (s *StakepooldManager) VoteVersion() (uint32, error) {
...
+	firstrun := true
+	for k, v := range responses {
+		if firstrun {
+			firstrun = false
+			lastResponse = v
+		}
+
+		if v.IsMine != lastResponse.IsMine ||
+			v.PubKeyAddr != lastResponse.PubKeyAddr {
+			vErr := fmt.Errorf("wallets %d and %d have different ValideAddress responses",
```
Comment (chappjc) [inline stakepooldclient/stakepooldclient.go]: would be good to log the mismatching fields (or just both fields from both responses)

File: stakepooldclient/stakepooldclient.go [hunk 3]
```diff
@@ -265,6 +266,48 @@ func (s *StakepooldManager) VoteVersion() (uint32, error) {
...
+		if err != nil {
+			log.Errorf("ValidateAddress RPC failed on stakepoold instance %d: %v", i, err)
+			return nil, err
+		}
+		responses[i] = resp
+	}
+
+	// Ensure responses are identical
+	var lastResponse *pb.ValidateAddressResponse
+	lastServer := 0
```
Comment (chappjc) [inline stakepooldclient/stakepooldclient.go]: Just a style thing - at least in dcrdata's codebase, we always init with `var name type` when the zero value is desired.  I'm not sure how consistent dcrstakepool is with variable initialization though.

**Pull request merged**